source/common/ssl/context_impl.cc

```diff
@@ -54,14 +54,33 @@ ContextImpl::ContextImpl(ContextManagerImpl& parent, Stats::Scope& scope,
 
   int verify_mode = SSL_VERIFY_NONE;
 
-  if (!config.caCertFile().empty()) {
-    ca_cert_ = loadCert(config.caCertFile());
-    ca_file_path_ = config.caCertFile();
-    // set CA certificate
-    int rc = SSL_CTX_load_verify_locations(ctx_.get(), config.caCertFile().c_str(), nullptr);
-    if (0 == rc) {
+  if (!config.caCert().empty()) {
+    ca_file_path_ = config.caCertPath();
```

Comment:
The only place this value is used is in ContextImpl::getCaCertInformation() (just to print the path of the cert, which is output by an admin handler).  Can we just get rid of it entirely?  It made sense when all config pointed to a path on disk; it makes less sense now and adds complication.

Reply:
It's also used in the error messages.